rotkehlchen/chain/ethereum/modules/nfts.py

```diff
@@ -221,8 +256,8 @@ def get_balances(
                     if exist_result is None or bool(exist_result[0]) is False:
                         cursor.execute(
                             'INSERT OR IGNORE INTO nfts('
-                            'identifier, name, last_price, last_price_asset, manual_price, owner_address'  # noqa: E501
-                            ') VALUES(?, ?, ?, ?, ?, ?)',
+                            'identifier, name, last_price, last_price_asset, manual_price, owner_address, is_lp, image_url, usd_price, collection_name'  # noqa: E501
+                            ') VALUES(?, ?, ?, ?, ?, ?, ?, ?, ?, ?)',
                             entry,
                         )
```

Comment:
Question: What happens if the user transfers an NFT? Do you know when the information is deleted from this table or if it ever gets deleted? Just asking because it came to my mind what we do in that case

Reply:
Hmmmmm no I guess we don't delete it. But it will make sense, good point!

Reply:
Please verify what we do now with this information. Before it was not a problem (at least that I know, check database integrity checks). Now since you query cache maybe you have to ignore or delete them